roundedimageview/src/main/java/com/makeramen/roundedimageview/RoundedTransformationBuilder.java

```diff
@@ -66,22 +67,55 @@ public RoundedTransformationBuilder cornerRadius(float radius) {
    *
    * @param corner the corner to set.
    * @param radius the radius in px.
-   * @return the builder for chaning.
+   * @return the builder for chaining.
    */
   public RoundedTransformationBuilder cornerRadius(int corner, float radius) {
     mCornerRadii[corner] = radius;
     return this;
   }
 
   /**
+   * Set radius to all corners, defined in chosen {@link Corners} enum
+   *
+   * @param corners the corners to set
+   * @param radius  the radius in px
+   * @return the builder for chaining
+   */
+  public RoundedTransformationBuilder corners(Corners corners, float radius) {
+    mCornerRadii = corners.getAll(radius);
+    return this;
+  }
+
+  /**
+   * Set radius to all corners in density independent pixels, defined in chosen {@link Corners} enum
+   *
+   * @param corners the corners to set
+   * @param radius  the radius in density independent pixels
+   * @return the builder for chaining
+   */
+  public RoundedTransformationBuilder cornersDp(Corners corners, float radius) {
+    mCornerRadii = corners.getAll(dp(radius));
+    return this;
+  }
+
+  /**
+   * Converts density independent pixels to normal pixels
+   *
+   * @param value the value to convert
+   * @return converted value
+   */
+  private float dp(final float value) {
```

Comment:
Let's rename this to something more clear. I thought it was converting px to dp at first, not the other way around. Perhaps `dpToPx()`